src/GeneratorV2/IndentedTextWriter.cs

```diff
@@ -46,10 +46,17 @@ public void WriteLine(string str)
 
         public void Flush() => Writer.Flush();
 
-        public Scope Indentation()
+        public Indentation Indent()
         {
             CurrentIndentation++;
-            return new Scope(this);
+            return new Indentation(this);
+        }
+
+        // Utility method for creating a cs scope "{}".
+        public Scope Scope()
```

Comment:
I think it might be better to be explicit here and say `CsScope` as the `Cs` prefix is also used in other parts of the code.